### DISCUSSION TITLE
Accessing the underlying HTMLElement

### DISCUSSION THREAD

**Discussion by yatesco** [ivanceras/sauron#50] [Q&A] [answered]
Hi - how do I access the actual underlying HTMLElement. Highcharts (for example) wants access to it. Thanks!

**Comment by ivanceras** [comment, marked as answer]
You can access it by assigning an Id to an element and access it using query selector or getElementById.

**Reply by yatesco** [reply]
thanks @ivanceras. Unfortunately this won't work as the number of charts will be dynamic.
